app/views/observations/form/_projects.html.erb

```diff
@@ -13,8 +13,7 @@
       <% @projects.each do |project| %>
         <%= check_box_with_label(form: f_p, field: :"id_#{project.id}",
                                  checked: @project_checks[project.id],
-                                 disabled: @observation.user != @user &&
-                                   !project.is_member?(@user),
+                                 disabled: project.check_box_disabled?(@observation, @user),
```

Comment:
Hmmm.  I thought maybe you named it `Project#check_box_disabled?` because of some Rails automagic that required that method to exist.  But if you're calling it explicitly, you could name it anything, right?

In that case, can we rename this so that it doesn't look like Project knows details of the create observation view?  I think it would also help because the way it is currently named I found myself asking "which check box?" when I first encountered it in Project.

Oh right, it's that one with the complicated logic.  Am I getting this right? -- The box is disabled if the project is not accepting observations, OR if the session user is not the owner of the observation and is not a member of the project...

Darn, I'm feeling dense right about now.  Wait, but it's only getting to the second clause if it is accepting observations, which means that it must be an open project (?) which means only the owner will ever see this check box, right?  Presumably I'm wrong, but what case(s) am I missing?  Can you help me understand the logic here?  I'm thoroughly confused.

Best to write the comment now while it's fresh in your mind! :)   (Than waiting for Nimmo to ask you what the hell you were thinking 10 years from now.  I know this from personal experience. :)

Reply:
Accepting Observations is separate from Open Membership (one of the realizations I had yesterday).  The logic about it being a different user is from before this PR so I was just getting logic out of the ERB file.  I agree the name sucks.  It was late.  I'll figure out a better name for it.

Reply:
Better name was easy once I flipped the logic around.